setup.py

```diff
@@ -3,11 +3,8 @@
 
 _version_file = 'hooked/version.py'
 
-if sys.version_info.major < 3:
-    execfile(_version_file)
-else:
-    with open(_version_file) as vf:
-        exec(vf.read())
+with open(_version_file) as vf:
+    exec(vf.read())
```

Comment:
Not sure if we want to use `exec` here as they mean different things in python2 vs python3: see [this](https://stackoverflow.com/questions/15086040/behavior-of-exec-function-in-python-2-and-python-3/15087355#15087355)